scripted-pipeline/src/main/resources/archetype-resources/pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>${groupId}</groupId>
+    <artifactId>${artifactId}</artifactId>
+    <version>${version}</version>
+    <name>Scripted pipeline</name>
+    <description>TODO</description>
+    <url>https://jenkins.io/doc/book/pipeline/syntax/#scripted-pipeline</url>
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>http://opensource.org/licenses/MIT</url>
+        </license>
+    </licenses>
+    <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <!-- Dependency versions -->
+        <groovy.version>2.4.3</groovy.version>
+        <junit.version>4.1</junit.version>
+        <jenkins-pipeline-unit.version>1.1</jenkins-pipeline-unit.version>
+    </properties>
+    <repositories>
+        <repository>
+            <id>repo.jenkins-ci.org</id>
+            <url>https://repo.jenkins-ci.org/public/</url>
```

Comment:
Does it actually need the Jenkins repos?

Reply:
Indeed there is no need.